Lib/test/test_cppext.py

```diff
@@ -31,6 +34,8 @@ def test_build_cpp03(self):
     @unittest.skipIf(
         '-fsanitize' in (sysconfig.get_config_var('PY_CFLAGS') or ''),
         'test does not work with analyzing builds')
+    # the test uses pip which needs a TLS connection to PyPI
+    @unittest.skipIf(ssl is None, 'No ssl module')
```

Comment:
It would be better to have a decorator like ``@test.support.requires_venv_with_pip``. requires_venv_with_pip() checks also for zlib and ctypes, so it's more complicated than just ssl in practice.

Reply:
We already have that decorator on this test.